packages/metacore/src/metaforms/interfaces.ts

```diff
@@ -144,6 +144,7 @@ export interface MultistringMetaProps extends CommonProps {
 
 export interface IThreeStateSwitch extends CommonProps {
   type: 'threeStateSwitch'
+  options?: [{ value: boolean | undefined; label: string }, { value: boolean | undefined; label: string }, { value: boolean | undefined; label: string }];
```

Comment:
co spis takhle:
```typescript
labels: {
    true?: string;
    false?: string;
    undefined?: string;
}
```
Prijde mi to kratsi a nemusi se uvadet ty `value: boolean | undefined` hodnoty, ktery jsou zbytecny

Reply:
Nechal bych takhle. 
Ty value nejsou zbytečný, můžeš ted díky tomu změnit pořadí významu tlačítek. 
A nepojmenovával bych propertu klíčovými slovy. Co myslíš ?

Reply:
ok